tools/make-boot-image

```diff
@@ -4,11 +4,11 @@ set -e
 
 TMP_DIR=""
 TMP_IMAGE=""
-IMAGE_SIZE=0
+IMAGE_SIZE=${IMAGE_SIZE:-0}
 MEGABYTE=$((1024 * 1024))
 BOOT_MOUNT=""
 LOOP=""
-NAME=$(basename $0)
+NAME="$(basename "$0")"
```

Comment:
Are these nested double-quotes really accepted?

Reply:
Yep, and it stops shellcheck complaining e.g. try

sh
echo "foo "${HOME}""

Reply:
Wow - and `echo "$(basename "$(echo "${HOME}")")"`!